src/frontend/js/utils/test/factories.ts

```diff
@@ -0,0 +1,17 @@
+import faker from 'faker';
+
+export const BlockMapFactory = (blocks: object[]) => {
+  const defaultBlock = {
+    key: faker.random.alphaNumeric(5),
```

Comment:
I saw that you are using `Draft.genKey()` elsewhere. So maybe you do not need to install faker just to generate a draft block key.

Reply:
It's bad I had already forgotten about it !